load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -36,6 +37,10 @@ import (
 	suppliertypes "github.com/pokt-network/poktroll/x/supplier/types"
 )
 
+// skipNotifyDownstreamSteps is a flag that indicates whether the downstream steps
+// should receive and process the current notification.
+const skipNotifyDownstreamSteps = true
```

Comment:
Not a fan of this. We're returning a local constant inside a mapping function.

If there was a bug in prod that we have to fix right away, I'd understand.

Here, we either need to:
1. Have a flag and make the flow functional
2. Remove it altogether and rethink how to work without it.

Reply:
This was intended to name the `skip` return of the `channel.Map`. Seems that both you and Bryan find this confusing. Will remove.